### DISCUSSION TITLE
[sudo] password for mastodon

### DISCUSSION THREAD

**Discussion by WebMagi** [mastodon/mastodon#17720] [Q&A] [answered]
Hi,

Does anyone have experience with managing a mastodon Droplet over on Digital Ocean? I used their Marketplace 1-Click App, see here: [https://marketplace.digitalocean.com/apps/mastodon](url)

I was initially using just an SSH Key to login to the Droplet Console, but I kept getting the following prompt after switching from "root" over to the "mastodon" system user, and then running simple commands like "sudo journalctl -u mastodon-web": 

**"[sudo] password for mastodon:"**

I have been unable to determine what password this prompt is expecting. I have front-end passwords for the admin and users, but I do not recall setting up a "mastodon" user password during server creation, or since.

As a test, I went ahead and created a root password, but this did not help. 

I am unable to run any Mastodon upgrade, because I can't get passed the above password prompt. Has anyone else run across this issue, or have an idea how I can resolve it?

Thanks so much!

**Comment by Gargron** [comment, marked as answer]
The mastodon user on the droplet does not have sudo access, and is not supposed to. Run `journalctl` and other system commands as root, or create a personal user with sudo privileges.

**Reply by data-88** [reply]
Thank you!

**Reply by TomBener** [reply]
Sorry I don’t understand this explanation fully. Currently I use the command below to restart my Mastodon server:

```shell
$ sudo systemctl restart mastodon-*
```

But after typing the command, it prompted to input the password. However, I connected DigitalOcean via a SSH key. I have no idea what the password is.

What command should be used to restart my Mastodon server to bypass the password? Thanks!

**Reply by schalkneethling** [reply]
@TomBener - If you did `sudo su - mastodon`. You will need to type `exit` to get back to the root user. From there you can run the `systemctl` commands. You can then run `sudo su - mastodon` to log back in as the mastodon user.

**Comment by WebMagi** [comment]
Thank you, that really helped!
